### PR TITLE
More 10081 fixes

### PR DESCRIPTION
**Pull request by garrettwrong**
Some changes required to allow the 10081 experiment to run to completion and use the recently added symmetry feature.

### CODE REVIEW

File: gallery/experiments/experimental_abinitio_pipeline_10081.py
Comment (j-c-c) [inline gallery/experiments/experimental_abinitio_pipeline_10081.py]: misplaced quote: `RelionSource(..., symmetry_group="C4, ...)"` --> `RelionSource(..., symmetry_group="C4", ...)`

File: gallery/experiments/experimental_abinitio_pipeline_10081.py
Comment (j-c-c) [inline gallery/experiments/experimental_abinitio_pipeline_10081.py:125]: Turns out that the `symmetry_group` pass-through was not implemented for `ClassAvgSource`s. Added the changes in PR #1151 to merge into this PR if everything looks good.

**Review by j-c-c — COMMENTED** [review]
It appears we weren't actually passing the symmetry groups into `ClassAvgSource`s. I added the necessary changes in #1151. If you're good with those changes I can merge them into this branch.

File: gallery/experiments/experimental_abinitio_pipeline_10081.py
Comment (garrettwrong) [reply gallery/experiments/experimental_abinitio_pipeline_10081.py]: fixed, thanks

**Comment by garrettwrong** [comment]
Added a diagnostic log message.

**Review by j-c-c — DISMISSED** [review]
Looks good!

**Comment by garrettwrong** [comment]
~I'll kick off re-running this~ I've started this run and will ensure the log message fires before merging into `develop`.

**Comment by garrettwrong** [comment]
Reconstruction completed.  Boosting message fired, good, but it was not formatted.  Just added the missing 'f'.  Will merge after the CI completes.

**Pull request merged**
